utils/index.js

```diff
@@ -19,6 +19,7 @@ exports.getReactScriptsVersion = function getReactScriptsVersion(cliVersion) {
       major: Number(semver.major(cliVersion)),
       minor: Number(semver.minor(cliVersion)),
       patch: Number(semver.patch(cliVersion)),
+      v: "" + semver.major(cliVersion) + semver.minor(cliVersion) + semver.patch(cliVersion)
```

Comment:
Two things:

1. I'd use template string interpolation here rather than implicit type coercion (less magical for other to read).
2. Please rename `v` to `concatenatedVersion`. We know just by reading the variable name what it is.

```suggestion
      concatenatedVersion: `${semver.major(cliVersion)}${semver.minor(cliVersion)}${semver.patch(cliVersion)}`
```

Reply:
thank you for the valuable feedback :) i changed the fix to your suggestion.